src/main/java/com/revature/repos/SurveyRepo.java

```diff
@@ -17,5 +17,11 @@
 
 	@Query("From Survey ORDER BY dateCreated DESC")
 	List<Survey> findAllOrderByDateCreatedDes();
+	
+	@Query("FROM Survey s WHERE s.template = true ORDER BY dateCreated DESC")
```

Comment:
try not to use query for this, make name findByTemplateOrderByDateCreatedDesc(boolean template) and it should work, then in your service layer you can pass in true or false, also might want to add pagination so perhaps Page<List<Survey>> findByTemplateOrderByDateCreatedDesc(boolean template, Pageable page)

Reply:
done